build-farm/platform-specific-configurations/solaris.sh

```diff
@@ -37,3 +37,7 @@ export PATH=/opt/solarisstudio12.3/bin/:/opt/csw/bin/:/usr/ccs/bin:$PATH:/usr/sf
 export LC_ALL=C
 export HOTSPOT_DISABLE_DTRACE_PROBES=true
 export ENFORCE_CC_COMPILER_REV=5.12
+
+# sepcial handling for jdk8 build "ant" which must use jdk8+
+# to match the version we installed by ansible
+export JDK8_BOOT_DIR=/usr
```

Comment:
Is that where we are storing the JDK 8 BOOT JDK? Seems like an odd location?

Reply:
i think in order to not care about the patch version, we create symblink to /usr/bin/java on Solaris

https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Solaris.yml#L278

Reply:
I'm with @karianna on this one - the current location defined in jenkins should be correct i.e. `/opt/csw/java/jdk1.8.0_181`. At the moment the machines have a symlink from `/usr/bin/java` to that location so I'm unclear on why that change is required. If we want to update it we could do it in jenkins, but I really don't want it forced in the build scripts like this.